xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/WinVideoFilter.cpp

```diff
@@ -228,7 +228,7 @@ bool COutputShader::Create(bool useLUT, bool useDithering, int ditherDepth, bool
 {
   m_useLut = useLUT;
   m_ditherDepth = ditherDepth;
-  m_toneMapping = toneMapping;
+  m_toneMapping = toneMapping && !DX::DeviceResources::Get()->Is10BitSwapchain();
```

Comment:
10bit swapchain doesn't mean that tonemapping should be disapbled always.

Reply:
Done